CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v0.19.1](https://github.com/buildkite/terraform-provider-buildkite/compare/v0.19.0...v0.19.1)
+### Fixes
+* Error: input: Clusters has not been activated for your organization [[Issue #287](https://github.com/buildkite/terraform-provider-buildkite/issues/287)] @mcncl
```

Comment:
We've been using the PR and title for this instead of the issue link. We can link back from the PR to the issue it closes, but its the PR that introduces the change that is logged here

Reply:
```suggestion
* SUP-1078 Fix input error for org not using Clusters [[PR #288](https://github.com/buildkite/terraform-provider-buildkite/pull/288)] @mcncl
```